src/contractor/contractor.cpp

```diff
@@ -307,7 +308,7 @@ int Contractor::Run()
 
     util::Log() << "Loading edge-expanded graph representation";
 
-    util::DeallocatingVector<extractor::EdgeBasedEdge> edge_based_edge_list;
+    std::vector<extractor::EdgeBasedEdge> edge_based_edge_list;
```

Comment:
This is not true anymore, since the vector is moved into the function (==destroyed for the outside world) and the scope takes care of the deallocation.
The only reason to have a `DeallocatingVector` is locations that don't deallocate by scope.